packages/flutter/test/material/range_slider_test.dart

```diff
@@ -1338,6 +1338,98 @@ void main() {
 
   });
 
+  testWidgets('Range Slider removes value indicator from overlay if Slider gets disposed without value indicator animation completing.', (WidgetTester tester) async {
+    final ThemeData theme = _buildTheme();
+    final SliderThemeData sliderTheme = theme.sliderTheme;
+    RangeValues values = const RangeValues(0.5, 0.75);
+
+    Widget buildApp({
+      Color activeColor,
+      Color inactiveColor,
+      int divisions,
```

Comment:
Looks like divisions isn't used, and is hardcoded to 3 below, was that intentional?

Reply:
Done